### DISCUSSION TITLE
how to generate 'lookup table' of the distortion by 'Gimp' or 'PhotoShop'

### DISCUSSION THREAD

**Discussion by TheXiong** [ImageMagick/ImageMagick6#217] [General]
### ImageMagick version

6

### Operating system, version and so on

windows 10

### Description

https://legacy.imagemagick.org/Usage/mapping/

As an image is used for the 'lookup table' of the distortion, you can create, or modify a the distortion map using an image editor, such as 'Gimp' or 'PhotoShop', giving you the freedom to do some really fancy and complex distortions.

**Comment by fmw42** [comment]
Perhaps you might show us some examples.
